projects/1.18/assets/ydms-wizarding-world-of-harry-potter/harrypotter/lang/zh_cn.json

```
@@ -0,0 +1,117 @@
+{
+"effect.harrypotter.wingardium_leviosa_effect": "羽加迪姆·勒维奥萨",
+"screen.harrypotter.hotbar": "快捷栏",
+"screen.harrypotter.spellbook": "法术书",
+"screen.harrypotter.expleft": "剩余经验：",
+"screen.harrypotter.plus": "+",
+"screen.harrypotter.lvl": "等级",
+"screen.harrypotter.accio": "飞来咒",
+"screen.harrypotter.aqua_eructo": "水涌喷射",
+"screen.harrypotter.ascendo": "上升咒",
+"screen.harrypotter.avada_kedavra": "阿瓦达索命",
+"screen.harrypotter.avis": "飞鸟群群",
+"screen.harrypotter.confringo": "霹雳爆炸",
+"screen.harrypotter.depulso": "速速退去",
+"screen.harrypotter.expecto_patronum": "呼神护卫",
+"screen.harrypotter.expelliarmus": "除你武器",
+"screen.harrypotter.fumos": "烟雾缭绕",
+"screen.harrypotter.glacius": "冰冻定身",
+"screen.harrypotter.herbivicus": "速速生长",
+"screen.harrypotter.incendio": "火焰熊熊",
+"screen.harrypotter.lumos": "荧光闪烁",
+"screen.harrypotter.melofors": "南瓜套头",
+"screen.harrypotter.mobilicorpus": "身体飘行",
+"screen.harrypotter.reparo": "修复如初",
+"screen.harrypotter.sectumsempra": "神锋无影",
+"screen.harrypotter.vulnera_sanentur": "速速愈合",
+"screen.harrypotter.wingardium_leviosa": "羽加迪姆·勒维奥萨",
+"screen.harrypotter.episkey": "愈合如初",
+"screen.harrypotter.alarte_ascandare": "烟消云散升天去",
+"screen.harrypotter.finite": "万咒皆终",
+"screen.harrypotter.evanesco": "消失无影",
+"screen.harrypotter.fire_storm": "火焰风暴",
+
+"screen.harrypotter.accio.description": "将一个物品拉过来。",
+"screen.harrypotter.aqua_eructo.description": "创造一股水柱。",
+"screen.harrypotter.ascendo.description": "将自己升到天上。",
+"screen.harrypotter.avada_kedavra.description": "瞬间死亡。",
+"screen.harrypotter.avis.description": "召唤一群鸟。",
+"screen.harrypotter.confringo.description": "使目标爆炸。",
+"screen.harrypotter.depulso.description": "将目标推开。",
+"screen.harrypotter.expecto_patronum.description": "召唤守护神。",
+"screen.harrypotter.expelliarmus.description": "缴械对手。",
+"screen.harrypotter.fumos.description": "产生防御性烟幕。",
+"screen.harrypotter.glacius.description": "冻结目标。",
+"screen.harrypotter.herbivicus.description": "使植物快速生长。",
+"screen.harrypotter.incendio.description": "召唤火焰。",
+"screen.harrypotter.lumos.description": "创造一个光源。",
+"screen.harrypotter.melofors.description": "戴上南瓜头。",
+"screen.harrypotter.mobilicorpus.description": "移动实体。",

```

```suggestion
"screen.harrypotter.mobilicorpus.description": "移动生物。",
```
这里原文不是`entities`，除非它能移动下落中的沙子（